src/palette.js

```diff
@@ -1472,3 +1472,5 @@ palette.SchemeType;
     palette.register(scheme);
   }
 })();
+
+export default palette;
```

Comment:
Maybe I dont see it but the code is transpiled with babel. I use that code locally and it works :)